### PR TITLE
Museum icon fix

### LINKED ISSUE #1280: Different museum icons
Museums starting to show on scale 1 km with old style icon. But when I zoom in to 200 m - icon changes to new one. Maybe app should use one icon?
Version: 2021.09.01-6-Google

Screenshot_20210910-122457~2
Screenshot_20210910-122519~2

### PR DESCRIPTION
**Pull request by endim8**
This PR fixes #1280

kvdsjksdkjfkfhvf

### CODE REVIEW

File: data/styles/clear/style-night/symbols/museum-m.svg [hunk 1]
```diff
@@ -1,16 +1,34 @@
...
    version="1.1"
    viewBox="0 0 24 24"
    height="24"
-   width="24">
+   width="24"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg">
+  <sodipodi:namedview
```
Comment (biodranik) [inline data/styles/clear/style-night/symbols/museum-m.svg:14]: This doesn't look right.

File: data/styles/clear/style-clear/symbols/museum-m.svg [hunk 2]
```diff
@@ -38,10 +37,25 @@
...
+     fill="none"
+     fill-rule="evenodd"
+     id="g920"
+     transform="matrix(1.7008547,0,0,1.7008547,-17.481481,-0.2694132)">
+    <path
+       fill="#ffffff"
+       d="m 13.333333,4.991453 4,-1.7777778 4,1.7777778 v 0.8888889 h -0.888889 v 4.4444441 h 0.888889 v 0.888889 h -8 v -0.888889 h 0.888889 V 5.8803419 h -0.888889 z m 2.133328,0.8888889 v 4.4444441 h 1.244445 V 5.8803419 Z m 2.577773,0 v 4.4444441 h 1.244444 V 5.8803419 Z"
+       id="path918" />
+  </g>
+  <metadata
```
Comment (biodranik) [inline data/styles/clear/style-clear/symbols/museum-m.svg:52]: It is not strictly necessary.

**Review by biodranik — COMMENTED** [review]
@pastk @vng please take a look

**Comment by endim8** [comment]
Moved to https://github.com/organicmaps/organicmaps/pull/2157

**Pull request closed without merging**